cypress/fixtures/registration_success.json

```diff
@@ -0,0 +1,14 @@
+{
+  "status": "success",
+  "data": {
+    "id": 37,
+    "provider": "email",
+    "uid": "user123@mail.com",
+    "allow_password_change": false,
+    "email": "user123@mail.com",
+    "created_at": "2020-10-20T14:32:41.566Z",
+    "updated_at": "2020-10-20T14:32:41.679Z",
+    "role": "registered",
+    "message": "Registration successful, now you have access to El-gaucho mobile app. Visit your appstore!"
```

Comment:
are you sending this message from the backend?

Reply:
No! We all got help from Oliver to 'hardcode' the sucessmessage and also set an errormessage.
Then I did the same for the error regarding the Loginform so that a errormessage displays there as well if user types in wron password